TestREADME.md

```diff
@@ -365,7 +365,7 @@ Response body:
 ```
 {
   "journeyType": "BUSINESS_VERIFICATION",
-  "origin": vat,
+  "origin": vatc,
```

Comment:
It is only a minor point but If you look in Kibana you will see that the origin is in upper case characters.